subhosting/manual/quick_start.md

```diff
@@ -1,4 +1,52 @@
-# Getting started with subhosting
+# Subhosting Quick Start
+
+Looking for the smallest possible example that shows how to deploy code to
+Deno's isolate cloud? We've got you covered below, or you can skip to the
+[more detailed getting started guide](#getting_started).
+
+```ts
+// 1.) Get API access info ready
+const accessToken = Deno.env.get("DEPLOY_ACCESS_TOKEN");
+const orgId = Deno.env.get("DEPLOY_ORG_ID");
+const API = "https://api.deno.com/v1";
+const headers = {
+  Authorization: `Bearer ${accessToken}`,
+  "Content-Type": "application/json",
+};
+
+// 2.) Create a new project
+const pr = await fetch(`${API}/organizations/${orgId}/projects`, {
+  method: "POST",
+  headers,
+  body: JSON.stringify({
+    name: null, // randomly generates project name
+  }),
+});
+const project = await pr.json();
+
+// 3.) Deploy a "hello world" server to the new project
+const dr = await fetch(`${API}/projects/${project.id}/deployments`, {
+  method: "POST",
+  headers,
+  body: JSON.stringify({
+    entryPointUrl: "main.ts",
+    assets: {
+      "main.ts": {
+        "kind": "file",
+        "content": `Deno.serve(() => new Response("Hello, World!"));`,
+        "encoding": "utf-8",
+      },
+    },
+    envVars: {},
+  }),
+});
+console.log(dr.status);
+```
+
+<br/>
+<a name="getting_started"></a>
+
+## Getting started with subhosting
 
 To get started with subhosting, you will need to create an organization in the
 [Deno Deploy dashboard](https://dash.deno.com/orgs/new). Follow the on-screen
```

Comment:
Can't comment on the exact line here since it wasn't part of the diff I guess but wanted to note that 

> Going through the onboarding flow, you will likely also generate an access token, which you will use to access the [REST API](https://deno-docs--subhosting-docs-update.deno.dev/subhosting/api/).

isn't entirely true. The token you create during onboarding only lasts for 1hr right now — which we did as a security provision for people who may make their way through onboarding a bit too hastily and not realize the power of what was generated for you — so we could look into changing this but as it stands now if you want to keep building you'll have to generate a permanent one.

Reply:
Going through the onboarding flow, you will likely also generate an **access
token**, which you will use to access the [REST API](../api/index.md). If you
didn't do this, or your token has expired, you can
[generate a new one here](https://dash.deno.com/account#access-tokens).

How's that sound?